mayastor/src/core/reactor.rs

```diff
@@ -288,7 +288,7 @@ impl Reactor {
             warn!("calling poll on a reactor who is not in the INIT state");
         }
 
-        if cfg!(debug_assertions) {
+        if std::env::var("DEVELOPER_DELAYED").is_ok() {
```

Comment:
Perhaps `MAYASTOR_DELAY` instead, for consistency, as `src/bin/spdk.rs` already uses that?

Reply:
Good idea :+1: